addon.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" ?><addon id="plugin.video.nrl-live" name="NRL Live" provider-name="Aussie Add-ons" version="0.0.1">
```

Comment:
Can you revert this commit? Our CI build script manage_repo.py takes care of it anyway so no need to bump :)

Reply:
Will do, change was only meant for a local copy of the addon to get rid of the addon update notification. Didn't intend to push the change.